datadog/resource_datadog_logs_archive_order_test.go

```diff
@@ -0,0 +1,95 @@
+package datadog
+
+import (
+	"context"
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"testing"
+)
+
+func ArchiveOrderConfig() string {
+	return fmt.Sprintf(`
+resource "datadog_logs_archive_order" "archives" {
+	archive_ids = []
```

Comment:
Can you create a test with real logs archive?

Reply:
I added a test to see if the archive_ids order from the state match with the real one.
If I would add archive order with real archive, I would mean get the archive from a real client and import the id before the test start, which seems not possible with the Test helper